angular-dfp.js

```diff
@@ -76,6 +81,13 @@ angular.module('ngDfp', [])
           }
         });
 
+	/**
+         Set the page targeting key->values
+         */
+        angular.forEach(pageTargeting, function (value, key) {
+          googletag.pubads().setTargeting(key,value);
```

Comment:
Sorry but the angular.forEach method works with the (value, key) arguments order, unless I use another forEach I won't be able t change the order of them even tho I agree that having them the other way would make the code look nicer. Docs: https://docs.angularjs.org/api/ng/function/angular.forEach

Want me to use a different forEach method that will keep the formatting? I think using the angular one is good as this is an Angular module.

Let me know.